### PR TITLE
build: enable strict mode.

### PR DESCRIPTION
**Pull request by phtaylor**

### CODE REVIEW

File: src/Renderer/GLBaseRenderer.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable guard-for-in */
+// @ts-ignore
 import throttle from 'lodash-es/throttle'
```
Comment (phtaylor) [inline src/Renderer/GLBaseRenderer.ts:3]: this line gave me a compiler error

Comment (mauroporras) [reply src/Renderer/GLBaseRenderer.ts:3]: Can you pull and try again? It's building on my side.

Screen Shot 2021-11-29 at 11 45 42

Comment (phtaylor) [reply src/Renderer/GLBaseRenderer.ts:3]: yes, builds for me too.

**Comment by phtaylor** [comment]
@mauroporras  can you merge and build the pre-release TC-strict mode package?

**Review by mauroporras — APPROVED** [review]

**Pull request merged**